controllers/argocd_metrics_controller.go

```diff
@@ -292,6 +308,79 @@ func (r *ArgoCDMetricsReconciler) createPrometheusRuleIfAbsent(namespace string,
 	return err
 }
 
+func (r *ArgoCDMetricsReconciler) createDashboardsIfAbsent(reqLogger logr.Logger) error {
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: dashboardNamespace}, &corev1.Namespace{})
+	if err != nil {
+		reqLogger.Info("OpenShift dashboards are not installed, skipping dashboard installation",
```

Comment:
could we rephrase this log statement? Could be a little confusing to users
maybe it could say something like "Monitoring dashboards are not supported on this cluster, skipping dashboard installation"? 

Lmk what you think